kasigazibrian/WeConnectReact
# #157834788 Create all the required components

### What does this PR do?
Adds all the components

### Description of Task to be completed?
Adds all the components that the user will interact with including the `css` and
`reactstrap` with `bootstrap 4` for redesigning the components.

### How should this be manually tested?
When you run the application, you should be able to view the home page component, login, signup
logout edit business, delete business components. Some of the components are displayed below.

<img width="1440" alt="screen shot 2018-06-15 at 12 45 34" src="https://user-images.githubusercontent.com/26734746/41462087-53dbe7bc-709a-11e8-994e-5466c1776b70.png">
<img width="1440" alt="screen shot 2018-06-15 at 12 45 47" src="https://user-images.githubusercontent.com/26734746/41462144-8bb74eb0-709a-11e8-9146-7ff57028f90f.png">
<img width="1440" alt="screen shot 2018-06-15 at 12 45 58" src="https://user-images.githubusercontent.com/26734746/41462241-db889084-709a-11e8-8ebb-f75d4e6d00e6.png">


### Any background context you want to provide?
Previously, these components were not available.

### What are the relevant pivotal tracker stories?
[#157834788](https://www.pivotaltracker.com/story/show/157834788)
